packages/frontend/src/components/ProjectConnection/DbtForms/DbtNoneForm.tsx

```diff
@@ -40,27 +40,6 @@ const DbtNoneForm: FC<{ disabled: boolean }> = ({ disabled }) => (
             </a>
             ) from your command line.
         </Callout>
-        <BooleanSwitch
```

Comment:
What is the reason to remove this input?

Reply:
I didn't see a reason to include the refresh-dbt button in this case since all it'll do is run a "Select 1" query and test the warehouse, which would be quite misleading for users.

Although now I'm wondering, is that actually `true` by default when I create on the backend?

Reply:
I think is `false` by default. The reason is that when the user creates a project via the CLI, they can see the indication that they need to update the project settings to be able to refresh from the UI.

Reply:
I think we should roll back this change and discuss this separately so it doesn't block the main fix.

Reply:
Done ✅